### PR TITLE
add page for 27.0 rc testing

### PR DESCRIPTION
**Pull request by glozow**

### CODE REVIEW

**Comment by glozow** [comment]
Thanks @cbergqvist @tdb3 @davidgumberg @marcofleon and fantastic testing guide! Let us know if there's anything you'd like to add.

File: _posts/2024-03-27-v27-rc-testing.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  release version.
+
+- To ensure that users don't experience issues with the new software, it's
+  essential that the rcs are thoroughly tested. This special review club
+  meeting is for people who want to help with that vital review process.
+
+- This [Bitcoin Core Release Candidate Testing
+  Guide](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/27.0-Release-Candidate-Testing-Guide) provides guidance for testing the release candidate.
+
+  - It is recommended to go through the "Preparation" steps ahead of the meeting, especially if you want to compile from source. Verify and confirm the successful installation of v27rc1.
```
Comment (cbergqvist) [inline _posts/2024-03-27-v27-rc-testing.md:30]: Maybe extend with a heads up:
```suggestion
  - It is recommended to go through the "Preparation" steps ahead of the meeting, especially if you want to compile from source. Verify and confirm the successful installation of v27rc1, as well as v25.1 and v26 (older versions are used in some of the tests). BDB support is required in one of the tests.
```

**Review by cbergqvist — COMMENTED** [review]
Thanks for helping out with this @glozow!

**Comment by davidgumberg** [comment]
LGTM 👍

**Pull request merged**